.github/workflows/netlify.yaml

```diff
@@ -0,0 +1,49 @@
+name: netlify production
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Can you include `develop` as well. It's free to deploy anyway, so we can observe how the site behaves in production